docs/src/tutorials/integrated/soil_canopy_tutorial.jl

```diff
@@ -273,50 +273,40 @@ canopy = ClimaLand.Canopy.CanopyModel{FT}(
 # Now we can combine the soil and canopy models into a single combined model.
 land = SoilCanopyModel{FT}(soilco2, soil, canopy);
 
-# Now we can initialize the state vectors and model coordinates, and initialize
-# the explicit/implicit tendencies as usual. The Richard's equation time
-# stepping is done implicitly, while the canopy model may be explicitly stepped,
-# so we use an IMEX (implicit-explicit) scheme for the combined model.
-
-Y, p, coords = initialize(land);
-exp_tendency! = make_exp_tendency(land);
-imp_tendency! = make_imp_tendency(land);
-jacobian! = make_jacobian(land);
-jac_kwargs =
-    (; jac_prototype = ClimaLand.FieldMatrixWithSolver(Y), Wfact = jacobian!);
-
 # We need to provide initial conditions for the model:
-Y.soil.ϑ_l = FT(0.4)
-Y.soil.θ_i = FT(0.0)
-T_0 = FT(288.7)
-ρc_s =
-    volumetric_heat_capacity.(
-        Y.soil.ϑ_l,
-        Y.soil.θ_i,
-        land.soil.parameters.ρc_ds,
-        earth_param_set,
-    )
-Y.soil.ρe_int =
-    volumetric_internal_energy.(Y.soil.θ_i, ρc_s, T_0, earth_param_set)
-
-Y.soilco2.C .= FT(0.000412) # set to atmospheric co2, mol co2 per mol air
-
-canopy_retention_model = canopy.hydraulics.parameters.retention_model
-ψ_stem_0 = FT(-1e5 / 9800)
-ψ_leaf_0 = FT(-2e5 / 9800)
-
-S_l_ini =
-    inverse_water_retention_curve.(
-        canopy_retention_model,
-        [ψ_stem_0, ψ_leaf_0],
-        plant_ν,
-        plant_S_s,
-    )
-for i in 1:2
-    Y.canopy.hydraulics.ϑ_l.:($i) .=
-        augmented_liquid_fraction.(plant_ν, S_l_ini[i])
+function set_ic!(Y, p, t0, model)
```

Comment:
(just seconding Julia's comment - here we pass in model, but use land, canopy, earth_param_set, etc, without getting them from `model`)